Description: MAN: Settings > About asks to donate to MAN (donate.mancorporation.com)
Author: MAN
---
--- a/panels/system/about/cc-about-page.blp
+++ b/panels/system/about/cc-about-page.blp
@@ -27,8 +27,8 @@
 
         Adw.PreferencesGroup {
           Adw.ActionRow {
-            title: _("Support GNOME");
-            subtitle: _("GNOME needs your help. Please donate today.");
+            title: _("Donate to MAN!");
+            subtitle: _("MAN is free, but it’s not free to maintain.");
 
             [suffix]
             Button {
--- a/panels/system/about/cc-about-page.c
+++ b/panels/system/about/cc-about-page.c
@@ -132,7 +132,7 @@
 static void
 on_donate_button_clicked_cb (CcAboutPage *self)
 {
-    g_app_info_launch_default_for_uri ("https://donate.gnome.org/", NULL, NULL);
+    g_app_info_launch_default_for_uri ("https://donate.mancorporation.com/", NULL, NULL);
 }
 
 static void
